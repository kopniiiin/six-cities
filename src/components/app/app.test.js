import React from "react";
import {shallow} from "enzyme";

import App, {testProps} from "./app.jsx";

describe(`snapshot test: App component`, () => {
  it(`should render correctly`, () => expect(
      shallow(<App {...testProps}/>)
  ).toMatchSnapshot());
});
