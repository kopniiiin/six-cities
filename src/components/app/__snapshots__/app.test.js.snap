// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: App component should render correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      exact={true}
      path="/"
    >
      <Main
        offers={
          Array [
            Object {
              "bedroomAmount": 4,
              "description": "Lots of paper",
              "features": Array [
                "paper",
                "coffee machine",
              ],
              "guestAmount": 4,
              "host": Object {
                "isPro": false,
                "name": "Host",
                "photo": Object {
                  "alt": "Host photo",
                  "src": "img/avatar-max.jpg",
                },
              },
              "id": "4",
              "isFavorite": false,
              "isPremium": false,
              "name": "Paper place",
              "photos": Array [
                Object {
                  "alt": "Place photo",
                  "src": "img/room.jpg",
                },
              ],
              "price": 4,
              "rating": 4,
              "type": "room",
            },
          ]
        }
        onOfferCardNameClick={[Function]}
      />
    </Route>
    <Route
      exact={true}
      path="/dev-offer"
    >
      <OfferScreen
        bedroomAmount={4}
        description="Lots of paper"
        features={
          Array [
            "paper",
            "coffee machine",
          ]
        }
        guestAmount={4}
        host={
          Object {
            "isPro": false,
            "name": "Host",
            "photo": Object {
              "alt": "Host photo",
              "src": "img/avatar-max.jpg",
            },
          }
        }
        id="4"
        isFavorite={false}
        isPremium={false}
        name="Paper place"
        photos={
          Array [
            Object {
              "alt": "Place photo",
              "src": "img/room.jpg",
            },
          ]
        }
        price={4}
        rating={4}
        type="room"
      />
    </Route>
  </Switch>
</BrowserRouter>
`;
